homeassistant/components/device_tracker/gpslogger.py

```diff
@@ -0,0 +1,73 @@
+"""
+Support for GPSLogger platform.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/device_tracker.gpslogger/
+"""
+import logging
+
+from homeassistant.const import HTTP_UNPROCESSABLE_ENTITY
+from homeassistant.components.http import HomeAssistantView
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['http']
+
+
+def setup_scanner(hass, config, see):
+    """Setup an endpoint for the GPSLogger application."""
+    hass.wsgi.register_view(LocationSet(hass, see))
+
+    return True
+
+
+class LocationSet(HomeAssistantView):
+    """View to handle locative requests."""
+
+    url = '/api/gpslogger'
+    name = 'api:gpslogger'
+
+    def __init__(self, hass, see):
+        """Initialize Location url endpoints."""
+        super().__init__(hass)
+        self.see = see
+
+    def get(self, request):
```

Comment:
This should be annotated with the `@callback` annotation found in `homeassistant.core`